release-notes.md

```diff
@@ -1,3 +1,11 @@
+# Quasar-17.9.0 #
+## Partner registration - Move all time consuming jobs to run in batch ##
+Cloning of UICONF and Category takes 4 seconds, moving them to batch
```

Comment:
write a more general description, the fact that it takes 4 sec is relevant to a specific copy case

Reply:
done